x-pack/solutions/observability/plugins/slo/public/hooks/use_fetch_apm_suggestions.ts

```diff
@@ -47,11 +47,12 @@ export function useFetchApmSuggestions({
             start: moment().subtract(2, 'days').toISOString(),
             end: moment().toISOString(),
             fieldValue: search,
-            ...(!!serviceName && fieldName !== 'service.name' && { serviceName }),
+            ...(!!serviceName &&
+              serviceName !== '*' &&
+              fieldName !== 'service.name' && { serviceName }),
```

Comment:
Let's use ALL_VALUE when possible, and you can remove the first check then.

```suggestion
            ...(serviceName !== ALL_VALUE &&
              fieldName !== 'service.name' && { serviceName }),
```